clusterDeployer.py

```diff
@@ -545,6 +693,14 @@ def update_etc_hosts(self) -> None:
         lh = host.LocalHost()
         lh.run("systemctl restart libvirtd")
 
+    def update_dnsmasq(self, *, setup: bool = True) -> None:
```

Comment:
I like it. Of course, it's mainly a style question and thus never "needed".

It forces the caller to use keyword arguments, which I think is often a better style (as the argument name is visible at the caller site). Otherwise, it's less obvious what `self.update_dnsmasq(False)` is supposed to mean.

Reply:
Ah, now that I understand what it does, I like it too. :)